src/clr-core/badge/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ngPackage": {
+    "lib": {
+      "entryFile": "badge.element.ts"
```

Comment:
Can we have this point to `index.ts`? Also in the package.json we need to add a new side effect entrypoint. `"./**/clr-core-badge.js"`

https://github.com/vmware/clarity/blob/master/src/clr-core/package.json#L11

Reply:
Yes and will change the tag's `package.json` as well.